tensorboard/plugins/core/core_plugin.py

```diff
@@ -451,17 +450,19 @@ def define_flags(self, parser):
 
   def fix_flags(self, flags):
     """Fixes standard TensorBoard CLI flags to parser."""
+    FlagsException = base_plugin.FlagsException
     if flags.inspect:
       if flags.logdir and flags.event_file:
-        raise ValueError(
+        raise FlagsException(
             'Must specify either --logdir or --event_file, but not both.')
       if not (flags.logdir or flags.event_file):
-        raise ValueError('Must specify either --logdir or --event_file.')
+        raise FlagsException('Must specify either --logdir or --event_file.')
     elif not flags.db and not flags.logdir:
-      raise ValueError('A logdir or db must be specified. '
-                       'For example `tensorboard --logdir mylogdir` '
-                       'or `tensorboard --db sqlite:~/.tensorboard.db`. '
-                       'Run `tensorboard --helpfull` for details and examples.')
+      raise FlagsException('A logdir or db must be specified.\n\n'
+                       'For example "tensorboard --logdir mylogdir"\n'
+                       'or `tensorboard --db sqlite:~/.tensorboard.db`.\n\n'
+                       'Run `tensorboard --helpshort` for a list of all options.\n'
+                       'Run `tensorboard --helpfull` for additional details and examples.\n')
```

Comment:
nit: These hanging indents have improper indentation. (Before this
change, they were all lined up at the opening parenthesis; after this
change, they’re just straggling.) Either indent them by four more spaces
to match, or change them to non-hanging indents:

```py
raise FlagsException(
    'A logdir or db must be specified.\n\n'
    'For example...'
)
```

I prefer the latter because it’s not brittle to changes that (like this
one) would require the whole block to be reflowed due to a change in
just one line, but the style guide permits either.

Reply:
Moving to a different PR.